spec/rspec/mocks/verifying_doubles/class_double_with_class_loaded_spec.rb

```diff
@@ -28,6 +28,16 @@ module Mocks
         prevents { expect(o).to receive(:defined_instance_method) }
       end
 
+      it "suggests using `instance_double` when an instance method is stubbed" do
+        o = class_double("LoadedClass")
+        prevents(/Perhaps you meant to use `instance_double`/) { allow(o).to receive(:defined_instance_method) }
+      end
+
+      it "doesn't suggest `instance_double` when a non-instance method is stubbed'" do
+        o = class_double("LoadedClass")
+        prevents(/(?!Perhaps you meant to use `instance_double`)/) { allow(o).to receive(:undefined_instance_method) }
```

Comment:
Wow, thanks for the feedback! This works now - the only way I departed was in extracting the constant. I used a slightly different name in each place so as to avoid generating a warning. Let me know if there's anything else I can do. Thanks!